bundle/compliance/cis_eks/rules/cis_3_2_2/test.rego

```diff
@@ -1,23 +1,24 @@
 package compliance.cis_eks.rules.cis_3_2_2
 
+import data.compliance.cis_eks.data_adapter
 import data.kubernetes_common.test_data
 import data.lib.test
 
 test_violation {
-	test.assert_fail(finding) with input as rule_input("")
-	test.assert_fail(finding) with input as rule_input("--authorization-mode AlwaysAllow")
-	test.assert_fail(finding) with input as rule_input_with_external("--authorization-mode AlwaysAllow", create_process_config("AlwaysAllow"))
-	test.assert_fail(finding) with input as rule_input_with_external("", create_process_config("AlwaysAllow"))
+	test.assert_fail(finding) with input as rule_input("") with data.benchmark_data_adapter as data_adapter
+	test.assert_fail(finding) with input as rule_input("--authorization-mode AlwaysAllow") with data.benchmark_data_adapter as data_adapter
+	test.assert_fail(finding) with input as rule_input_with_external("--authorization-mode AlwaysAllow", create_process_config("AlwaysAllow")) with data.benchmark_data_adapter as data_adapter
+	test.assert_fail(finding) with input as rule_input_with_external("", create_process_config("AlwaysAllow")) with data.benchmark_data_adapter as data_adapter
 }
```

Comment:
we can change split into two parts so we'll need to use `with data.benchmark_data_adapter as data_adapter` only once:
```rego
violations {
	test.assert_fail(finding) with input as rule_input("")
	test.assert_fail(finding) with input as rule_input("--read-only-port 10")
	test.assert_fail(finding) with input as rule_input_with_external("--read-only-port 10", create_process_config(0))
	test.assert_fail(finding) with input as rule_input_with_external("", create_process_config(10))
}

test_violations {
	violations with data.benchmark_data_adapter as data_adapter
}
```